docs/src/domains/steps/tls.md

```diff
@@ -1,85 +1,91 @@
-or using the [command line interface](../../administration/cli/_index.md).
+To add your custom certificate, follow these steps:
 
 {{< codetabs >}}
 
 ---
-title=In the Console
+title=Using the CLI
 file=none
 highlight=false
 ---
 
-- Select the project where you want to add a certificate.
-- Click {{< icon settings >}} **Settings**.
-- Click **Certificates**.
-- Click **+ Add**.
-- Fill in your private key, public key certificate, and (optionally) intermediate SSL certificates.
-- Click **Add Certificate**.
+1. Add the certificate with the following command:
+
+    <!-- This is in HTML to get the variable shortcode to work properly -->
+    <div class="highlight"><pre class="chroma"><code class="language-bash" data-lang="bash">platform domain:add {{<variable "DOMAIN" >}} --cert {{<variable "PATH_TO_CERTIFICATE_FILE" >}} --key {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}</code></pre></div>
+
+    For example:
+
+    ```bash
+    platform domain:add secure.example.com --cert /etc/TLS/private/secure-example-com.crt --key /etc/TLS/private/secure-example-com.key
+    ```
+
+    You can optionally include intermediate SSL certificates by adding <code>--chain {{<variable "PATH_TO_FILE" >}}</code> for each one.
+
+2. Redeploy your production environment with the following command:
+
+    ```bash
+    platform environment:redeploy
+    ```
 
 <--->
+
 ---
-title=Using the CLI
+title=In the Console
 file=none
 highlight=false
 ---
 
-Run the following command:
-
-```bash
-platform domain:add -p <PROJECT_ID> <DOMAIN> --cert <PATH_TO_CERTIFICATE_FILE> --key <PATH_TO_PRIVATE_KEY_FILE>
-```
+1. Open the project where you want to add a certificate.
+2. Click {{< icon settings >}} **Settings**.
+3. Click **Certificates**.
+4. Click **+ Add**.
+5. Fill in your private key, public key certificate, and (optionally) intermediate SSL certificates.
+6. Click **Add Certificate**.
+7. Access your production environment.
+8. Click {{< icon more >}} **More**.
+9. Click **Redeploy**.
 
-For example:
-
-```bash
-platform domain:add -p abcdefg123456 secure.example.com --cert /etc/TLS/private/secure-example-com.crt --key /etc/TLS/private/secure-example-com.key
-```
+{{< /codetabs >}}
 
-You can optionally include intermediate SSL certificates by adding `--chain <PATH_TO_FILE>` for each one.
+### Change the private key format
 
-{{< /codetabs >}}
+Your certificate's private key needs to be in PKCS #1 format, which means it starts with `-----BEGIN RSA PRIVATE KEY-----`.
+If it has `-----BEGIN PRIVATE KEY-----` instead, it's in PKCS #8 format and you need to change it.
 
-For the new certificate to be taken into account, you need to [redeploy the environment](../../development/troubleshoot.md#force-a-redeploy).
+To convert the private key from PKCS #8 to PKCS #1 format, run the following command:
 
 ```bash
-platform environment:redeploy
+openTLS rsa -in {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}.key -out {{<variable "PATH_TO_PRIVATE_KEY_FILE" >}}.rsa.key
```

Comment:
:lady_beetle: Generally, the path to a file includes the extension, so this way of presenting it might be slightly confusing.